mullvad-cli/src/main.rs

```diff
@@ -138,11 +138,19 @@ enum Cli {
     #[clap(subcommand)]
     CustomList(custom_list::CustomList),
 
-    /// Apply a JSON patch
+    /// Apply a JSON patch generated by 'export-settings'
+    #[clap(arg_required_else_help = true)]
     ImportSettings {
         /// File to read from. If this is "-", read from standard input
         file: String,
     },
+
+    /// Export a JSON patch based on the current settings
+    #[clap(arg_required_else_help = true)]
+    ExportSettings {
+        /// File to write to. If this is "-", write to standard output
+        file: String,
+    },
 }
```

Comment:
I know this is what the linear issue specified, but why write to a file as default? Is it not easy enough to pipe the output to a file?

Reply:
Discussed AFK.